core/client/app/styles/layouts/settings.css

```diff
@@ -34,7 +34,7 @@
     cursor: move;
 }
 
-.gh-blognav-url .fake-placeholder {
+.gh-blognav-url {
```

Comment:
I think this rule should be deleted. The specified colour is for the placeholder element that's no longer present so applying the colour to the entire input instead means that it has to be overridden in a higher-specificity rule (which it is so it's not noticeable but better to keep things clean! :smile:).